### ISSUE TITLE
multiple audios problem

### ISSUE THREAD

**Issue by yosgi** [webgameone/vue2audioComponent#1]
if there are multiple audios in page the currentProgress have bug 
its better to change the window.querySelector('audio') to this.$el.querySelector('audio')

**Comment by webgameone** [comment]
yes,you're right,Because of my project is multiple audios,so I did somthing else
